glib/glib.go

```diff
@@ -107,6 +107,8 @@ const (
 	TYPE_VARIANT   Type = C.G_TYPE_VARIANT
 )
 
+var TYPE_SOCKET Type = Type(C.G_TYPE_SOCKET)
```

Comment:
add this into the const group above please. There is no reason that this should be a var.

Reply:
It can't be treated as const in Go, because it is function call in C - it's ok for C macro, but not for Go.

Reply:
If it differs from the other constant types then please add a documentation comment describing why it has to be treated differently

Reply:
done